classes/class-wc-connect-stripe.php

```diff
@@ -46,6 +46,11 @@ public function get_oauth_url( $return_url = '' ) {
 			if ( empty( $return_url ) ) {
 				$return_url = admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe' );
 			}
+
+			if ( substr( $return_url, 0, 8 ) !== 'https://' ) {
+				return new WP_Error( 'invalid_url_protocol', __( 'Your site requires SSL in order to connect your Stripe account via WooCommerce Services', 'woocommerce-services' ) );
```

Comment:
Nitpick: I think saying `HTTPS` will be easier to understand than `SSL` for a merchant.